### ISSUE TITLE
Unable to reproduce Full Multinex while Multinex-Nano reproduces successfully on LOL-v1

### ISSUE THREAD

**Issue by lucky-star0** [albrateanu/multinex#11]
Hi authors,

Thank you for releasing the code and pretrained models.

I am currently reproducing Multinex on LOL-v1. Multinex-Nano can be reproduced successfully using the released configuration, while I am unable to reproduce the reported performance of the Full Multinex model using the current `Options/Multinex_LOL-v1.yaml`.

I have also confirmed that I am using the updated MS-SSIM implementation:

```python
def multiscale_ssim_loss(y_true, y_pred, max_val=1.0):
    return 1.0 - ms_ssim(
        y_true,
        y_pred,
        data_range=max_val,
        size_average=True,
        weights=[0.99, 0.009, 0.001],
        win_size=5
    )
```

While comparing the Full and Nano configurations, I noticed a difference in the effective learning-rate behavior.

For Full Multinex:

```yaml
optim_g:
  lr: 2e-4

scheduler:
  type: CosineAnnealingRestartCyclicLR
  periods: [46000, 104000]
  restart_weights: [1, 1]
  eta_mins: [0.0003, 0.000001]
```

According to the current implementation of `CosineAnnealingRestartCyclicLR`, the first cycle appears to change the learning rate approximately as:

```text
2e-4 -> 3e-4
```

during the first 46K iterations, because the first `eta_min` is higher than the base learning rate.

After that, the learning rate restarts to approximately `2e-4` and then decays toward `1e-6`.

In contrast, Multinex-Nano uses:

```yaml
optim_g:
  lr: 4e-4

scheduler:
  periods: [40000, 60000]
  restart_weights: [1, 1]
  eta_mins: [0.0003, 0.000001]
```

so its first cycle behaves as a normal decay:

```text
4e-4 -> 3e-4
```

before restarting and decaying toward `1e-6`.

Since Multinex-Nano reproduces successfully in the same environment while the Full model does not, I would like to clarify whether the Full-model learning-rate configuration is exactly the one used to train the released pretrained checkpoint.

Could you please clarify:

1. Was the released `Multinex_LOL-v1.pth` trained using exactly the current `Multinex_LOL-v1.yaml`?
2. Is the first Full-model cycle intentionally designed to increase the learning rate from `2e-4` to `3e-4`?
3. What exact learning-rate schedule was used to obtain the reported Full Multinex result?
4. Was `val_freq=5` used when selecting the reported best checkpoint?

Thank you for your help.
